app/infrastructure/common/utils/decorators/error_handler.py

```
@@ -13,16 +13,16 @@ def decorator(function: any) -> any:
         async def wrapper(*args, **kwargs):
             try:
                 return await function(*args, **kwargs)
-            except ScraperException as e:
+            except AppException as e:
                 logger.error("SCRAPER_EXCEPTION", exc_info=e)
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)